docs/src/examples.md

```diff
@@ -0,0 +1,140 @@
+Examples
+========
+
+The [`K8sClusterManager`](@ref) is intended to be used inside a [_Pod_](https://kubernetes.io/docs/concepts/workloads/pods/)
+running on a Kubernetes cluster.
+
+## Launching an interactive session
+
+The following manifest will create a Kubernetes [_Job_](https://kubernetes.io/docs/concepts/workloads/controllers/job/)
+named "interactive-session". This _Job_ will spawn a _Pod_ (see `spec.template.spec`) which will
+run an interactive Julia session with the latest release of K8sClusterManagers.jl installed.
+Be sure to create the required [_Service Account_ and associated permissions](../patterns/#required-permissions)
+before proceeding.
+
+````@eval
+using Markdown
+Markdown.parse("""
+```yaml
+$(read("interactive-session.yaml", String))
+```
+""")
+````
+
+To start the _Job_ and attach to the interactive Julia session you can run the following:
+
+```sh
+# Executed from the K8sClusterManager.jl root directory
+kubectl apply -f docs/src/interactive-session.yaml
+
+# Determine the generated pod name from the job
+manager_pod=$(kubectl get pods -l job-name=interactive-session --output=jsonpath='{.items[*].metadata.name}')
+echo $manager_pod
+
+# Attach to the interactive Julia session running in the pod.
+# Note: You may need to wait for K8sClusterManagers.jl to finish installing
+kubectl attach -it pod/${manager_pod?}
+```
+
+### Launching workers
+
+Once you've attached to the interactive session you can use [`K8sClusterManager`](@ref) to
+spawn k8s workers. For our example we'll be using a small amount of CPU/Memory to ensure
```

Comment:
Looks like "K8s" is preferable: https://github.com/cncf/foundation/blob/master/style-guide.md#2-kubernetes

Bonus: cube-control!